### ISSUE TITLE
Failed to get git context: couldn't parse origin url: first path segment in URL cannot contain colon

### ISSUE THREAD

**Issue by dombarnes** [Bearer/bearer#1459]
## Description & Reproduction
Azure DevOps git SSH Repos are fail initial scan.

Example Git Remote URL: `mycompany@vs-ssh.visualstudio.com:v3/mycompany/Project/RepoName`
Command: `bearer scan .`
## Expected Behavior

Bearer proceeds with scan

## Actual Behavior

```
Analyzing codebase
⠙  (149710/-) [7s]
Error: failed to get git context: couldn't parse origin url: parse "mycompany@vs-ssh.visualstudio.com:v3/mycompany/Project/Repo-Name": first path segment in URL cannot contain colon
```

## Your Environment

- Operating System and version: macOS 14.2.1
- Output of 'bearer version':
```
bearer version 1.34.0, build e142a5d5f37be48246e84fcd59ebff3cdc9f0bb5
```

**Comment by cfabianski** [comment]
Thanks @dombarnes for the report and sorry for the inconvenience!
We will be looking into it shortly. 
🐻 with us

**Comment by gmontard** [comment]
Hello @dombarnes, 

I'm happy to report that the fix is part of the 1.35 version that just go released:
https://github.com/bearer/bearer/releases/tag/v1.35.0
